src/NuGet.Core/NuGet.Commands/NuGet.CommandLine/Verbosity.cs

```diff
@@ -0,0 +1,12 @@
+// Copyright (c) .NET Foundation. All rights reserved.
```

Comment:
Shouldn't be used ideally.

Reply:
moving type back to nuget.commandline --- and translating while creating args.